app/javascript/components/Waybill.tsx

```diff
@@ -26,16 +26,19 @@ const Waybill = ({ currentUserRole }) => {
       componentMounted.current = false;
     };
   }, []);
-  const update_checkpoint_status = (id) => {
-    httpClient.route.get_routes(id).then((response) => setCheckpoints(response.data));
-
+  const update_checkpoint_status = (id, checkpoint) => {
```

Comment:
U get route in WaybillTable.tsx & may be, the best way to move this there? Because it doesn't need stay here.
![image](https://user-images.githubusercontent.com/60444425/163548627-c7a6d94a-553c-4e73-848c-6dda4e28da9a.png)